joomla/joomla-cms
# [4.2] Convert site status to service provider

### Summary of Changes
Convert the site status plugin to service provider.

### Testing Instructions
- Create three site status tasks for each type one
- Run the task
- Check the site status if it reflects the task type

### Actual result BEFORE applying this Pull Request
All works.

### Expected result AFTER applying this Pull Request
All works.
